### PR TITLE
Add APNS views

### PR DESCRIPTION
**Pull request by LilyFirefly**

### CODE REVIEW

**Comment by LilyFirefly** [comment]
@KevinEtienne @maxpeterson Review?

File: rest_framework_push_notifications/tests/test_serializers.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        fields = serializers.APNSDevice().fields.keys()
+        self.assertEqual(expected, set(fields))
+
+    def test_registration_id(self):
+        serializer = serializers.APNSDevice(data={'registration_id': 'test_id'})
+        self.assertTrue(serializer.is_valid())
+
+    def test_registration_id_required(self):
+        serializer = serializers.APNSDevice(data={})
+        self.assertFalse(serializer.is_valid())
```
Comment (kevinetienne) [inline rest_framework_push_notifications/tests/test_serializers.py:18]: maybe you should check for the error keys explicitly?

**Comment by kevinetienne** [comment]
Not sure if you need it but you could have a device list view.

**Comment by LilyFirefly** [comment]
Updated

**Pull request merged**